src/components/v2/Table/styles.ts

```diff
@@ -82,5 +86,13 @@ export const useStyles = () => {
         color: ${active ? theme.palette.interactive.success : theme.palette.text.primary};
       }
     `,
+    cellWrapper: css`
+      max-width: 115px;
```

Comment:
we've agreed to use theme.spacing to express values like this

Reply:
removed